docs/Screensharing.md

```diff
@@ -1,5 +1,19 @@
 # Screensharing
 
+ScreenSharing enables displaying streaming screen content shared by the Visitor
+during an Engagement.
+
 See [explanation video][0]
 
+## Setup
+
+- An iOS device running iOS 11.0+
+- No active Screencast or Airplay enabled on the device
+- Site configured to accept Visitor's screen
+
+## Annotations
+
+When there is an active Engagement the Operator can drawn simple shapes on the screen which will be translated
```

Comment:
Wording is a bit clumsy, what about

`When Operator is engaged with a Visitor and the Visitor has shared their screen, Operator can draw on the _screensharing_ view and the drawing will be drawn on the Visitor screen overlaying the application.`

Also, does this drawing work for arbitrary applications or only the one using our SDK?

Reply:
Only our sdk

Reply:
Can you specify that in the documentation?